CRM/Upgrade/Incremental/php/FiveSeventySix.php

```diff
@@ -57,4 +60,29 @@ public static function create_mesage_header_token() {
     return TRUE;
   }
 
+  /**
+   * Drop tables, disable the message template as they relate to event carts.
+   *
+   * It would be nice to delete the message template but who knows there could be a gotcha.
+   *
+   * @return bool
+   * @throws \CRM_Core_Exception
+   */
+  public static function removeEventCartAssets(): bool {
+    try {
+      CRM_Core_DAO::executeQuery("UPDATE civicrm_msg_template SET is_active = 0 WHERE workflow_name = 'event_registration_receipt'");
+      if (!CRM_Core_DAO::singleValueQuery('SELECT id FROM civicrm_events_in_carts LIMIT 1')) {
```

Comment:
@eileenmcnaughton is this duplicated in the below if ?

Reply:
One is "carts" and one is "in carts".

Reply:
wonderful